src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -295,7 +298,8 @@ class LedgerMaster
     // The passed lock is a reminder to callers.
     bool newPFWork(const char *name, std::unique_lock<std::recursive_mutex>&);
 
-private:
+    boost::optional<LedgerIndex> minSqlSeq() const;
```

Comment:
> Did you intend to lose the `private:` with this change? I'm guessing that was accidental...

Yes, private (and no subsequent public or protected) is already set on line 259.

Reply:
Good point.  Sorry for the noise.

Reply:
> `minSqlSeq()` is only called from one place, and that is in LedgerMaster.cpp. I'm thinking you could turn this into a lambda in the (admittedly large) function where it is called. Possibly better would be to make `minSqlSeq()` a file scoped function in the .cpp file. It might look like this:
> 
> ```
> // Helper function for LedgerMaster::doAdvance()
> static boost::optional<LedgerIndex>
> minSqlSeq(Application& app)
> {
>     boost::optional<LedgerIndex> seq;
>     *app.getLedgerDB().checkoutDb()
>         << "SELECT MIN(LedgerSeq) FROM Ledgers", soci::into(seq);
>     return seq;
> }
> ```

fixed